test/e2e/framework/util.go

```diff
@@ -259,6 +259,7 @@ func nowStamp() string {
 }
 
 func log(level string, format string, args ...interface{}) {
+	strings.TrimSuffix(format, "\n")
```

Comment:
This doesn't mutate `format`, but instead returns a new `string` with the suffix trimmed. You'll need to pass the returned value to `fmt.Fprintf`.

Reply:
Also, we shouldn't trim, vs. find and fix offenders in the logs.

Reply:
👍

Reply:
@timothysc can you clarify "find and fix offenders in the logs", I want to make sure I completely understand your review suggestion. Thank you! :smile:

Reply:
He means we should go through all the e2e code and remove any trailing newlines from log statements.

Reply:
Thank you for the clarification @ncdc!